src/main/java/uk/gov/di/ipv/lambda/CredentialIssuerHandler.java

```diff
@@ -24,8 +24,9 @@
         implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
 
     private final CredentialIssuerService credentialIssuerService;
+    private final ConfigurationService configurationService;
 
-    private final CredentialIssuers credentialIssuers;
+    private CredentialIssuers credentialIssuers = null;
```

Comment:
I think we don't actually need to explicitly set this to null here. Maybe it's better to though.

Reply:
I agree, don't need to set it explicitly.

Reply:
@SunilDGDS If you whip this `null` out I'll give the PR a 👍